sql/migrations/reports/000003_alter_reports_remove_source_batch_id.down.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE reports
+ADD COLUMN IF NOT EXISTS source_batch_id text DEFAULT '',
+ADD COLUMN IF NOT EXISTS source_task_id text DEFAULT '';
```

Comment:
Minor: What is the point of this migration?

Reply:
we have the down migrations everywhere though never used in production. Just following the standard thing in the code